deploy/rp-development.json

```diff
@@ -13,6 +13,9 @@
         },
         "rpServicePrincipalId": {
             "type": "string"
+        },
+        "storageAccountName": {
```

Comment:
Adding the SA to rp-development.json will break its callers and we're not using it anyway as there are no RP VMs in dev.  Not worth the hassle.  Please fix up pkg/deploy/generator so that rp-development.json is left untouched.